src/test/java/io/streams/e2e/flink/sql/SqlJobRunnerST.java

```diff
@@ -82,11 +87,25 @@ void testFlinkSqlRunnerSimpleFilter() {
         KubeResourceManager.getInstance().createOrUpdateResourceWithWait(
             KafkaTemplate.defaultKafka(namespace, "my-cluster")
                 .editSpec()
-                .withCruiseControl(null)
-                .withKafkaExporter(null)
+                .editKafka()
+                .editFirstListener()
+                .withAuth(new KafkaListenerAuthenticationScramSha512())
+                .endListener()
+                .endKafka()
```

Comment:
I think it would be better to build the listener completely to easily see how it looks like